.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -3,6 +3,16 @@ name: Bug report
 about: Create a report to help us improve
 ---
 
+**What part of Carbon does this bug correlate to?**
+* [ ]  All graphs (axes, legends etc)
```

Comment:
Should `All graphs` and `Graph attributes (axes, legends, regions...etc.)` be separate items

Reply:
I updated it here: https://github.com/cerner/carbon-graphs/pull/294/commits/c383611c2ea9c23eefe88de68ac8a41503f4a2d9